validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/Eth1VotingPeriod.java

```diff
@@ -14,30 +14,35 @@
 package tech.pegasys.teku.validator.coordinator;
 
 import static tech.pegasys.teku.pow.api.Eth1DataCachePeriodCalculator.calculateEth1DataCacheDurationPriorToFollowDistance;
-import static tech.pegasys.teku.util.config.Constants.EPOCHS_PER_ETH1_VOTING_PERIOD;
-import static tech.pegasys.teku.util.config.Constants.ETH1_FOLLOW_DISTANCE;
-import static tech.pegasys.teku.util.config.Constants.SECONDS_PER_ETH1_BLOCK;
-import static tech.pegasys.teku.util.config.Constants.SLOTS_PER_EPOCH;
 
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
-import tech.pegasys.teku.util.config.Constants;
+import tech.pegasys.teku.spec.SpecProvider;
+import tech.pegasys.teku.spec.constants.SpecConstants;
 
 public class Eth1VotingPeriod {
 
   private final UInt64 cacheDuration;
+  private final SpecProvider specProvider;
 
-  public Eth1VotingPeriod() {
+  public Eth1VotingPeriod(final SpecProvider specProvider) {
     cacheDuration = calculateEth1DataCacheDurationPriorToFollowDistance();
+    this.specProvider = specProvider;
```

Comment:
nit: I'd suggest swapping these two lines so `this.specProvider` is set first.   Generally setting fields before calling functions is good (admittedly it is a static function so won't matter but I had to go and check that to feel safe).